src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -61,6 +90,128 @@ class EmojiPickerMenu extends Component {
         if (this.props.forwardedRef && _.isFunction(this.props.forwardedRef)) {
             this.props.forwardedRef(this.searchInput);
         }
+
+        // Setup keypress/mouse handlers only if we have a keyboard (and not a touchscreen)
+        if (!canUseTouchScreen() && document) {
+            this.keyDownHandler = (e) => {
+                // Highlight the appropriate emoji
+                if (e.key.startsWith('Arrow')) {
+                    this.highlightAdjacentEmoji(e.key);
+                    this.toggleArrowKeysOnSearchInput();
+                }
+
+                // Select the highlighted emoji if enter is pressed
+                if (e.key === 'Enter' && this.state.highlightedIndex !== -1) {
+                    this.props.onEmojiSelected(this.state.filteredEmojis[this.state.highlightedIndex].code);
+                }
+            };
+            document.addEventListener('keydown', this.keyDownHandler);
+
+            // Re-enable pointer events and hovering over EmojiPickerItems when the mouse moves
+            this.mouseMoveHandler = () => {
+                if (this.state.shouldDisablePointerEvents) {
+                    this.setState({shouldDisablePointerEvents: false});
+                }
+            };
+            document.addEventListener('mousemove', this.mouseMoveHandler);
+        }
+    }
+
+    componentWillUnmount() {
+        // Cleanup all mouse/keydown event listeners that we've set up
+        if (!canUseTouchScreen() && document) {
+            document.removeEventListener('keydown', this.keyDownHandler);
+            document.removeEventListener('keydown', this.mouseMoveHandler);
+            this.toggleKeysOnSearchInput([], ['ArrowDown', 'ArrowLeft', 'ArrowRight', 'ArrowUp']);
+        }
+    }
+
+    /**
+     * Highlights emojis adjacent to the currently highlighted emoji depending on the arrowKey
+     * @param {String} arrowKey
+     */
+    highlightAdjacentEmoji(arrowKey) {
+        // If nothing is highlighted and an arrow key is pressed
+        // select the first emoji
+        if (this.state.highlightedIndex === -1) {
+            this.setState(prevState => ({highlightedIndex: prevState.filteredEmojis.length === emojis.length ? 8 : 0}));
```

Comment:
Why 8? Sorry, I just don't know what it means. Maybe leaving a comment about it would be nice.

Reply:
Ohh, it's the name of columns, you should use `numColumns` instead of hardcoding the number

Reply:
Good catch! I completely missed this but this is definitely what I was trying to do.